dev/README.md

```diff
@@ -149,7 +321,7 @@ and to check the logs, run:
 docker-compose -f dev/docker-compose/docker-compose.yml logs
 ```
 
-## Start the Fleet Manager
+### Start the Fleet Manager
 
 The [manager-run.sh](bin/manager-run.sh) script starts the Fleet Manager with all the configurations.
```

Comment:
To be updated? I don't think the script is named `manager-run.sh` anymore (same comment for the other links)